src/components/custom/tooltip-wrapper.jsx

```
@@ -0,0 +1,52 @@
+import { TooltipProvider, Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip';
+import { Arrow } from '@radix-ui/react-tooltip';
+import { useState } from 'react';
+import { Button } from '../ui/button';
+
+export function TooltipWrapper({ children, content, side = 'bottom', className, ...props }) {
```

ESLint

[error] 'children' is missing in props validation

(react/prop-types)